build-tools-internal/src/main/groovy/elasticsearch.bwc-test.gradle

```diff
@@ -50,5 +51,5 @@ plugins.withType(InternalJavaRestTestPlugin) {
   }
 }
 
-tasks.matching { it.name.equals("check") }.configureEach {dependsOn(bwcTestSnapshots) }
-tasks.matching { it.name.equals("test") }.configureEach {enabled = false}
+tasks.matching { it.name.equals("check") }.configureEach { dependsOn(bwcTestSnapshots) }
```

Comment:
We should convert these to use the new `named(Spec)` methods.

Reply:
We can't as it would fail since we don't necessarily have those tasks task in all projects that use bwc.